_includes/join_us.html

```diff
@@ -27,15 +27,6 @@ <h2>Support Black Python Devs</h2>
             script.id = "commitchange-script";
             script.src = "https://us.commitchange.com/js/donate-button.v2.js";
 
-            script.onload = function () {
```

Comment:
This wasn't executing before the A11y testing so this was moved out of this script